sample_size/multiple_testing.py

```diff
@@ -0,0 +1,67 @@
+from typing import List
+
+import numpy as np
+from statsmodels.stats.multitest import multipletests
+
+from sample_size.metrics import BaseMetric
+
+
+class MultipleTestingMixin:
+    """
+    This class calculates sample size required under the case of multiple testing
+
+    Attributes:
+    metrics: a list of BaseMetric registered by users
+    variants: number of variants, including control
+    alpha: statistical significance
+    power: statistical power
+
+    """
+
+    metrics: List[BaseMetric]
+    alpha: float
+    power: float
+    variants: int
+
+    REPLICATION: int = 500
+
+    def get_multiple_sample_size(self, lower: float, upper: float, depth: int = 0) -> int:
```

Comment:
nit - probably can add a docstring for readability - what each parameter (lower/upper) means for example. not easy to know lower and upper represent "sample size" when reading directly